Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -7275,6 +7277,7 @@ def apply_dns_signature_policy_command(args: dict) -> CommandResults:
                    f'<entry name="{edl}"><packet-capture>{packet_capture}</packet-capture>'
                    f'<action><{action}/></action></entry>'
                    f'</lists>'
+                   f'<sinkhole><ipv4-address>{ipv4_address}</ipv4-address><ipv6-address>{ipv6_adderss}</ipv6-address></sinkhole>'
```

Comment:
@samuelFain don't you need this **only** in case the action is sinkhole? and not in **every** case?

Reply:
Do we have a TPB to make sure that all other commands pass successfully with the additional parameters' default values?

Reply:
There is a TPB that executing this command, but it is not among the TPBs that run during the build. I ran it separately and it passed successfully.
Also, it's important to note that these default values are sent with every command use-case, and they do not impact any other use-case of this command except when `action=sinkhole` is used. We have already tested this specific case to ensure its functionality.